format-specs/geoparquet.md

```diff
@@ -80,10 +80,10 @@ which itself implements the model of
 Apart from the difference of encodings, the semantics are intended to match
 WKT2:2019, and a CRS in one encoding can generally be represented in the other.
 
-If CRS is not provided, all coordinates in the geometry must use longitude, latitude
-based on the WGS84 datum to store their data.
+If the `crs` field is not provided, the default CRS is [OGC:CRS84](https://www.opengis.net/def/crs/OGC/1.3/CRS84),
```

Comment:
```suggestion
If the `crs` field is not provided, all coordinates in the geometries must use longitude, latitude based on the WGS84 datum,
```

Reply:
all coordinates in the geometries must use longitude, latitude based on the WGS84 datum

Reply:
```suggestion
all coordinates in the geometries must use longitude, latitude based on the WGS84 datum
```